src/analytics/src/main/scala/osmesa/analytics/oneoffs/StreamingAOIMonitor.scala

```diff
@@ -0,0 +1,176 @@
+package osmesa.analytics.oneoffs
+
+import cats.data.{Validated, ValidatedNel}
+import cats.implicits._
+import com.monovore.decline.{CommandApp, Opts}
+import geotrellis.vector._
+import org.apache.spark.sql.SparkSession
+import org.apache.spark.sql.functions.{explode, from_unixtime, to_timestamp, udf}
+import org.locationtech.geomesa.spark.jts.{udf => _, _}
+import org.locationtech.jts.{geom => jts}
+import org.locationtech.jts.geom.prep._
+import osmesa.analytics.Analytics
+import vectorpipe.sources.Source
+
+import java.net.URI
+
+// These only required for test getAreaIndex implementation
+import geotrellis.vector.io._
+import geotrellis.vector.io.json._
+import spray.json._
+
+object StreamingAOIMonitor extends CommandApp(
+  name = "streaming-aoi-monitor",
+  header = "Streaming AOI Monitor",
+  main = {
+    val intervalOpt =
+      Opts
+        .option[String]("interval", help = "Period of time to aggregate over (d=daily, w=weekly, m=monthly)")
+        .validate("Must be one of 'd', 'w', or 'm'") { arg: String => arg.length == 1 && "dwm".contains(arg.apply(0)) }
+
+    val databaseUrlOpt =
+      Opts
+        .option[URI](
+        "database-url",
+        short = "d",
+        metavar = "database URL",
+        help = "URL of database containing AOI table (default: DATABASE_URL environment variable)"
+      )
+      .orElse(Opts.env[URI]("DATABASE_URL", help = "The URL of the database"))
+
+    val augmentedDiffSourceOpt =
+      Opts
+        .option[URI](
+        "augmented-diff-source",
+        short = "a",
+        metavar = "uri",
+        help = "Location of augmented diffs to process"
+      )
+
+    val startSequenceOpt =
```

Comment:
In the interest of Ya Aint Gonna Need It, the specs for our initial MVP are that the start / end sequences necessary are generated from the interval parameter passed by the scheduled task. Given that, we should probably remove these two sequence params and just generate the start and end sequence values from the interval, using `now` as the end.

Reply:
For testing purposes, it makes the most sense to leave these in for the time being.  I agree that we will ultimately draw the sequence numbers from different sources, but for now, being able to pull a known set of diffs is useful.

Reply:
Works for me for now. Given that though, could we add the logic that converts an interval to start/end sequence values now? If interval is provided, it would override any values passed to startSequence and endSequence or vice versa.

Reply:
The way I see it, the interval doesn't only affect the selection of start/stop sequences, but also how the data are aggregated.  (That was what that abortive comment was going to get at before I saw something shiny and forgot to finish typing.)  I want to have the control during testing to pick a window of time to look at, and also the freedom to select how it will be presented.  Or so the thinking goes.

Reply:
Ok. It makes sense to keep the start/stop params in for now, but I'm not sure what that means for the comment above though in regards to implementing the interval param. Are you saying that there's no value in doing that at all for this PR? If so, I don't really see how that implementation would change substantively as we go along seeing as its the implementation we need for this CLI to work as expected.

Reply:
I'm not 100% sure what you're getting at here.  I was putting up some incomplete work to foster discussion about what the output should look like.  We don't yet have an idea of what each interval's report should look like, which I think is the most important thing to figure out at this point.  I assume that there will be some kind of match/case that will process the grouped values differently based on the interval, but aside from that, we're lacking in the substance of those aggregations.  This initial commit was not a full and opinionated statement, just a rough structure so that we can stub out the remaining functionality.

In that vein, can we shift this discussion to what we want to see come out of the back end of this pipe—what data should be delivered to the subscriber for each interval?  Then I think we can fill in the provided armature.

Reply:
Yeah, the general stub here is certainly sufficient in terms of getting at some AugmentedDiffs and putting them in a DataFrame. No issues there. 

> about what the output should look like

Makes sense. For output, we need to end up with data in a structure that can be passed to a method that [sends emails via SES](https://docs.aws.amazon.com/ses/latest/DeveloperGuide/send-using-sdk-java.html). 

So maybe that's something like `List[Notification]` where `Notification` is something like:
```scala
// type aliases only to give meaning to base types
type Username = String
type EditCount = Int
case class Notification(userId: UUID, aoiName: String, start: DateTime, end: DateTime, geom: Geometry, counts: [(UserName, EditCount)])
```

???

If github comments are too low bandwidth, we can definitely chat offline. LMK.

Reply:
This was decided offline:

Seq[AugmentedDiff] + Seq[Feature[Geom, Data]] (from DB) --> Seq[email, aoiName, interval, Seq[(User, Info)]] where email, aoiName come from Data in DB and the rest is generated from this task.